open-craft/opencraft
# Enable forum digests

This PR enables and set up the forum digest feature.

It also fixes a problem where bulk email would fail to send because it wasn't getting properly rewritten to an email address verified in AuthSMTP.

**Testing**

Deploy a new instance using code in this PR.

Bulk email:

1. [Enable Bulk email](http://edx.readthedocs.io/projects/open-edx-release-notes/en/latest/bulk_email.html).
1. Log into LMS as staff.
1. Go to Instructor -> Email.
1. Send an email to yourself.
1. Check that the email was successfully delivered to your mailbox.

Forum notifications:

1. Log into LMS.
1. Go to the Discussion tab; you should see the Discussion homepage.
1. Tick the "Receive updates" box.
1. Make some new forum posts.
1. You should receive a digest email in 24 hours (you can also log into the VM and run `python manage.py forums_digest` as the `notifier` user to manually trigger the digest email task).

**Reviewers**
